packages/common/src/compiler/translator.test.ts

```diff
@@ -311,4 +313,15 @@ describe('convert tables from dbt models', () => {
             ),
         ).toStrictEqual(LIGHTDASH_TABLE_WITH_GROUP_BLOCK);
     });
+
+    it('should convert dbt model with metrics in meta and keep model spotlight override', () => {
```

Comment:
can you add some use cases for categories? 
And could we consider (for later) an invalid categories test as well?